plugins/updater/guest-js/index.ts

```diff
@@ -39,12 +39,15 @@ type DownloadEvent =
   | { event: "Progress"; data: { chunkLength: number } }
   | { event: "Finished" };
 
+class DownloadedBytes extends Resource {}
```

Comment:
I apologize but looks like a new class is not needed after all and we can just use `Resource` class directly instead.